### ISSUE TITLE
gips tries to fetch the unfetchable

### ISSUE THREAD

**Issue by ircwaves** [Applied-GeoSolutions/gips#41]
Reported by @bhbraswell :+1: 
```shell
icooke@rio:~$ gips_process landsat -p ndvi8sr -d 2013-182 -t 016025 -v4 --fetch
GIPS Data Processing (v0.8.2)
Retrieving inventory for site tiles
Fetching SR 016025 2013-07-01
===> Downloading: LC80160252013182LGN00.tar.bz
```
The bug is that `gips.data.landsat.landsatData.fetch` needs to check that the asset being requested is `DN`.   The above `ndvi8sr` needs the USGS Surface Reflectance (`SR`) asset, seeing that it is not there, the fetch is incorrectly downloading the `DN` asset.

I checked with Lindsay, and her recollection is that LC8SR is a manual download.  And as the fetch is employing devseed's landsat-util wrapper around S3 and Google's L8 stashes, it appears that there's not a SR fetch through those:
https://github.com/developmentseed/landsat-util/blob/develop/landsat/downloader.py#L206-L242

**Comment by ircwaves** [comment]
bump!

```
gips_inventory landsat -t 118063 -d 2016-05-10,2016-05-030  --fetch  -v5
GIPS Data Inventory (v0.8.2)
Retrieving inventory for site tiles
Fetching DN 118063 2016-05-17
===> Downloading: LC81180632016138LGN00.tar.bz
   100%    782.8 MiB      12.6 MiB/s            0:00:00 ETA    
     stored at /titan/data/landsat/stage
Reformatting bz->gz
removing /titan/data/landsat/stage/LC81180632016138LGN00.tar.bz
Fetching SR 118063 2016-05-17
===> Downloading: LC81180632016138LGN00.tar.bz
   100%    782.8 MiB      10.6 MiB/s            0:00:00 ETA    
     stored at /titan/data/landsat/stage
Reformatting bz->gz
```

**Comment by ircwaves** [comment]
fixed by #96
